receiver/mongodbatlasreceiver/logs.go

```diff
@@ -0,0 +1,242 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbatlasreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbatlasreceiver"
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"go.mongodb.org/atlas/mongodbatlas"
+	"go.opentelemetry.io/collector/component"
+)
+
+// combindedLogsReceiver wraps alerts and log receivers in a single log receiver to be consumed by the factory
+type combindedLogsReceiver struct {
+	alerts *alertsReceiver
+	logs   *receiver
+}
+
+type resourceInfo struct {
+	Org      *mongodbatlas.Organization
+	Project  *mongodbatlas.Project
+	Cluster  *mongodbatlas.Cluster
+	Hostname string
+	LogName  string
+}
+
+const collection_interval = time.Minute * 5
```

Comment:
Can we add a comment here with a link to the docs that calls out the 5 minute interval recommended? Also, should probably add this to the documentation in the readme.